src/Umbraco.Web.UI.Client/src/assets/lang/en.ts

```diff
@@ -1256,6 +1256,12 @@ export default {
 	colorpicker: {
 		noColors: 'You have not configured any approved colours',
 	},
+	colorPickerConfigurations: {
+		colorsTitle: 'Colors',
+		colorsDescription: 'Add, remove or sort colors',
+		showLabelTitle: 'Include labels?',
+		showLabelDescription: 'Stores colors as a Json object containing both the color hex string and label, rather than just the hex string.',
```

Comment:
```suggestion
		colorsTitle: 'Colours',
		colorsDescription: 'Add, remove or sort colours',
		showLabelTitle: 'Include labels?',
		showLabelDescription: 'Stores colours as a JSON object containing both the colour hex string and label, rather than just the hex string.',
```

As I understand, this file is for UK English, so "colors" should be "colours".